package.json

```diff
@@ -83,16 +83,16 @@
     "knex": "^0.95.6",
     "lodash": "^4.17.21",
     "moment": "^2.29.1",
-    "mssql": "^7.1.0",
+    "mssql": "^7.1.3",
     "nunjucks": "3.2.x",
     "nunjucks-date-filter": "^0.1.1",
     "passport": "^0.4.1",
     "passport-saml": "^2.2.0",
-    "sass": "^1.34.0",
-    "tedious": "^11.0.8",
+    "sass": "^1.35.1",
+    "tedious": "^11.0.9",
     "underscore": "^1.13.1",
-    "util": "^0.12.3",
+    "util": "^0.12.4",
     "validator": "^13.6.0",
-    "wmt-probation-rules": "ministryofjustice/wmt-probation-rules.git#v1.0.6"
+    "wmt-probation-rules": "github:ministryofjustice/wmt-probation-rules#v1.0.6"
```

Comment:
Does this mean we no longer need git installed on the docker image?

Reply:
no, still need git unfortunately. looks like this is a convention from npm (this happened when I ran npm upgrade)